README.md

```diff
@@ -68,3 +68,49 @@ Advanced courses that require prior knowledge in machine learning and AI.
 * [Mathematics of Machine Learning Summer School 2019 (University of Washington)](https://www.youtube.com/playlist?list=PLTPQEx-31JXhguCush5J7OGnEORofoCW9) | [Course Website](http://mathofml.cs.washington.edu/)
 
 * [Probabilistic Graphical Models (Carneggie Mellon University)](https://www.youtube.com/playlist?list=PLoZgVqqHOumTY2CAQHL45tQp6kmDnDcqn) | [Course Website](https://sailinglab.github.io/pgm-spring-2019/)
+
+* [Probabilistic and Statistical Machine Learning 2020 (University of Tübingen)](https://www.youtube.com/playlist?list=PL05umP7R6ij1tHaOFY96m5uX3J21a6yNd)
+
+* [Statistical Machine Learning 2020 (University of Tübingen)](https://www.youtube.com/playlist?list=PL05umP7R6ij2XCvrRzLokX6EoHWaGA2cC)
+
+* [Deep Learning (with PyTorch) 2020 (NYU)](https://www.youtube.com/playlist?list=PLLHTzKZzVU9eaEyErdV26ikyolxOsz6mq)
+
+* [Mobile Sensing and Robotics 2019 (Bonn University)](https://www.youtube.com/playlist?list=PLgnQpQtFTOGQJXx-x0t23RmRbjp_yMb4v)
+
+* [Sensors and State Estimation Course 2020 (Bonn University)](https://www.youtube.com/playlist?list=PLgnQpQtFTOGQh_J16IMwDlji18SWQ2PZ6)
+
+* [Photogrammetry 2015 (Bonn University)](https://www.youtube.com/playlist?list=PLgnQpQtFTOGRsi5vzy9PiQpNWHjq-bKN1)
+
+* [Applied Machine Learning 2020 (Columbia)](https://www.youtube.com/playlist?list=PL_pVmAaAnxIRnSw6wiCpSvshFyCREZmlM)
```

Comment:
Yeah it's a lot more practical than cs229 and mostly covers scikit learn. It's a nice supplement to the theoretical courses because it spends more time on things like model evaluation and calibration.